.travis.yml

```diff
@@ -0,0 +1,6 @@
+language: node_js
+node_js:
+  - "6"
```

Comment:
Why don't we bump this to v7 atleast? v8 will soon be LTS and v7 brings some important upgrades like `async` and `await`.